build/3-apillon-api.md

```diff
@@ -1158,3 +1158,714 @@ curl --location --request GET "https://api.apillon.io/hosting/websites/:websiteU
+                "id": 7,
+                "status": 5,
+                "chainId": 1287,
+                "transactionType": 1,
+                "transactionStatus": 1,
+                "transactionHash": "0xb988c8cda7ec8b441611b208360e0aace9c294e1ca5672375b21e815890a54d1",
+                "updateTime": "2023-06-13T10:15:58.000Z"
+            }
+        ],
+        "total": 1
+    }
+}
+```
+
+</CodeGroupItem>
+</CodeGroup>
+  </div>
+</div>
+
+##### Statuses
+
+| Number | Description         |
+|--------|---------------------|
+| 1      | Draft               |
+| 2      | Incomplete          |
+| 5      | Active              |
+| 8      | Marked for deletion |
+| 9      | Deleted             |
+
+##### Transaction Types
+
+| Number | Description                 |
+|--------|-----------------------------|
+| 1      | Deploy Contract             |
+| 2      | Transfer Contract Ownership |
+| 3      | Mint NFT                    |
+| 4      | Set Collection Base URI     |
+| 5      | Burn NFT                    |
+
+##### Transaction Status
+
+| Number | Description |
+|--------|-------------|
+| 1      | Pending     |
+| 2      | Confirmed   |
+| 3      | Failed      |
+| 4      | Error       |
+
+
+
+### Create NFT Collection
+
+> API that creates NFT collection and deploys it on selected network.
+
+#### POST /nfts/collections
+
+<div class="split_content">
+	<div class="split_side">
+
+#### Body fields
+
+| Name              | Type      | Description                                                                       | Required |
+|-------------------|-----------|-----------------------------------------------------------------------------------|----------|
+| project_uuid      | `string`  | Apillon project id.                                                               | true     |
+| chain             | `number`  | Blockchain id on which you want to release your collection.                       | true     |
+| symbol            | `string`  | NFT collection symbol (usually 3-4 characters long).                              | true     |
+| name              | `string`  | NFT collection name.                                                              | true     |
+| description       | `string`  | NFT collection description.                                                       | false    |
+| maxSupply         | `number`  | Maximal number of NFTs ever in existence.                                         | true     |
+| mintPrice         | `number`  | Price of NFT at mint stage.                                                       | true     |
+| baseUri           | `string`  | Base URI for collection metadata (token id and file extension is appended to it). | false    |
+| baseExtension     | `string`  | File extension that is auto appended after token id to form a full URL.           | true     |
+| isDrop            | `boolean` | Determines if collection is mintable by public.                                   | true     |
```

Comment:
I would add a warning in the description that if this is false that mintPrice and reserve and drop start are not used. Maybe even prepend all the names relevant with `drop`.

Reply:
Would it make sense to rename this to `isMint` instead of `isDrop`? Drop sounds like airdrop in which case NFTs are dropped to user wallets, mint requires users to connect their own wallet and send TX.